src/declarations/sent-tx-store.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
```

Comment:
This is a feature specific code. Please keep it within the folder of the feature.

Reply:
I have removed `eslint-disable`